Documentation/report.md

```diff
@@ -281,7 +281,18 @@ kubectl -n $METERING_NAMESPACE get reportgenerationqueries namespace-memory-requ
 }
 ```
 
-### Report Status
+### Inputs
+
+The `inputs` field of a Report `spec` can be used to pass custom values into a `ReportGenerationQuery`. It is an array of name-value pairs:
```

Comment:
- Probably just use the term `list` rather than `array`. It's less jargon.
- Put the second sentence on it's own line.

We probably don't want to merge this until later since it really has minimal use right now and it probably needs more examples of how it connects to a real query. That's why I didn't document it yet; it's very difficult to see how this is used without having a built-in query that supports custom inputs. ReportingStart/ReportingEnd are also not the best example since they're a bit more special case.